app/models/video.rb

```diff
@@ -1,9 +1,19 @@
 class Video < ActiveRecord::Base
   belongs_to :category
+  has_many :reviews, -> { order "created_at DESC" }
+  has_many :queue_items
 
   validates_presence_of :title, :description
 
   def self.search_by_title(search_term)
     search_term.blank? ? [] : where("title ILIKE ?", "%#{search_term}%").order(created_at: :desc)
```

Comment:
if `.where` does not find anything, it should return `[]` empty array/relation proxy.